Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "tauri-plugin-sql"
 version = "0.0.0"
-authors = [ "Tauri Programme within The Commons Conservancy" ]
```

Comment:
Is there any way to avoid extraneous changes like this in Cargo files? I don't remember making any changes here but I'm guessing there's some setting that formats this which I'm unaware of.